app/controllers/questionnaire_controller.rb

```diff
@@ -72,10 +72,10 @@ def check_content_hash
   end
 
   def set_response
-    invitation_token = InvitationToken.find_by_token(questionnaire_params[:q])
-    check_invitation_token(invitation_token)
+    the_response = Response.find_by_uuid(questionnaire_params[:uuid])
```

Comment:
Trouwens het feit dat je hier uuid hebt ipv q betekent toch ook dat je uuid ipv q voor de externe route (die we in de sms linkjes gebruiken) gaat gebruiken? Dat lijkt me wat onhandig. De reden dat we q gebruikten is omdat het 1 letter is, en iedere letter telt in onze SMS-invites waarin we deze links zetten. Ik zou zeggen maak er dan u van ipv q, als het maar niet meer letters wordt. een statische https://app.u-can-act.nl/?uuid=asdfbsdf waarbij uuid nooit verandert is verspilling van characters.

Reply:
Nee, het gaat zo:

linkje?q=fjakldsjas -> `token_controller`
logged in? Zo ja, redirect naar `/questionnaire/`
`questionnaire` -> vind een uuid of een mentor dashboard 
uuid -> `questionnaire/<UUID>`